lib/format.test.js

```diff
@@ -77,3 +77,30 @@ test('for a report with an error message with a location', t => {
 	};
 	t.deepEqual(actual, expected, 'it should return the correct location');
 });
+
+test('for a report with fixes', t => {
+	const input = {
+		results: [
+			{
+				messages: [
+					{
+						fix: {
+							range: [0, 0],
+							text: ';'
+						}
+					}
+				]
+			}
+		]
+	};
+
+	const [{solutions: [actual]}] = format(editor)(input);
+	const expected = {
+		position: {
+			end: {column: 0, row: 0},
+			start: {column: 0, row: 0}
+		},
+		replaceWith: ';'
+	};
+	t.like(actual, expected, 'it should return the correct location');
+});
```

Comment:
Unsure how useful a test like this is.. I think the additional assertions to the spec are fine